tests/test_api_build.py

```diff
@@ -1442,6 +1442,9 @@ def test_overlinking_detection_ignore_patterns(
     rm_rf(dest_bat)
 
 
+@pytest.mark.serial
+@pytest.mark.skipif(sys.platform == "linux", reason="DID NOT RAISE Over Depending Error")
+@pytest.mark.skipif(sys.platform == "darwin", reason="DID NOT RAISE Over Depending Error")
```

Comment:
It looks like you accidentally deleted the wrong files, we want to keep `_overdepending_detection`’s `build.bat` & `build.sh` that previously existed and delete `_overlinking_detection`’s autogenerated `bld.bat` & `build.sh`